The upstream build system uses symlinks to perform the build in the linux/
directory: this causes make to always fail the very first time it is called,
because apparentely it can't satisfy a dependency with a file created
during the same run.

This patch removes the symlink creation, since we make a copy of the needed
files in debian/rules.
Index: scrotwm/linux/Makefile
===================================================================
--- scrotwm.orig/linux/Makefile	2009-03-23 22:21:01.000000000 +0100
+++ scrotwm/linux/Makefile	2009-03-23 22:21:09.000000000 +0100
@@ -7,10 +7,7 @@
 
 LVERS= $(shell . ../lib/shlib_version; echo $$major.$$minor)
 
-all: symlink scrotwm libswmhack
-
-symlink:
-	ln -sf ../scrotwm.c ../lib/swm_hack.c .
+all: scrotwm libswmhack
 
 scrotwm: scrotwm.o linux.o
 	$(CC) $(LDFLAGS) -o $@ $+ $(LDADD)
